organization/app/organization/[id]/view-organization/page.tsx

```
@@ -0,0 +1,559 @@
+              {organization.name}
+            </h1>
+          )}
+        </div>
+
+        <div className="flex gap-3 w-full md:w-auto">
+          {isEditing ? (
+            <>
+              <button
+                onClick={() => setIsEditing(false)}
+                className="flex-1 md:flex-none flex items-center justify-center gap-2 px-5 py-2.5 rounded-lg bg-gray-100 hover:bg-gray-200 text-gray-800 transition disabled:opacity-50"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)